pkgs/misc/screensavers/physlock/default.nix

```diff
@@ -1,13 +1,11 @@
-{ stdenv, fetchFromGitHub }:
+{ stdenv, fetchurl  }:
 
 stdenv.mkDerivation rec {
-  version = "git-20150126";
-  name = "physlock-${version}";
-  src = fetchFromGitHub {
-    owner  = "muennich";
-    repo   = "physlock";
-    rev    = "b64dccc8c22710f8bf01eb5419590cdb0e65cabb";
```

Comment:
I am afraid that _without_ fetchFromGitHub the hash of the tarball is unstable over time.